docs/governance/politeia.md

```diff
@@ -39,8 +39,8 @@ For now, the process is something like this:
 3. Valid proposals appear publicly on Politeia, open for discussion, but voting does not begin immediately.  	
 4. Ticket-voting interval (~1 week) can be triggered by the proposal owner. Until ticket-voting is triggered, the proposal can be edited by its owner in response to community feedback.
 5. Ticket-voting starts, ticket-holders vote through their wallet (details tbc).
```

Comment:
Could remove "(details tbc)" here and add some details - like "Every ticket that was live 256 blocks before voting started is eligible to vote on the proposal. Votes by eligible tickets can be cast any time within the voting interval. Once votes are cast they cannot be changed."

Reply:
That wording sounds good to me. Used and pushed to branch.